client/src/components/ForumComment/ForumComment.css

```diff
@@ -0,0 +1,34 @@
+.ForumComment {
+    position: absolute;
+    left: 0%;
+    right: 0%;
+    top: 62.58%;
+    bottom: 0%;
+    
+    background: #C4C4C4;
```

Comment:
There's a --gray-dark variable for this now. So, you can go ahead and replace it.